src/main/java/uk/co/codezen/maven/redlinerpm/mojo/AbstractRpmMojo.java

```diff
@@ -108,6 +108,12 @@
     protected boolean performCheckingForExtraFiles = true;
 
     /**
+     * Whether to replace already existing artifacts
+     */
+    @Parameter
+    protected boolean shouldReplaceArtifact = false;
```

Comment:
Shouldn't it be `true` by default in order to follow the standard maven behavior?